### PR TITLE
Removing the clone from shellphish/afl-other-arches

### PR DESCRIPTION
**Pull request by g0kkk**
Have removed the clone of AFL-other-arch and instead clones AFL repo and apply patches directly onto it. Also, there is a fix for qemu not working in ubuntu 18.04. @zardus I guess this should do it.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,27 +8,39 @@
...
 AFL_MULTI_CGC_FUZZ  = os.path.join(AFL_MULTI_CGC_INSTALL_PATH)
 
+
 def _setup_other_arch():
-    # grab the afl-other-arch repo
+    # revisiting the afl mirrorer repo
     if not os.path.exists(AFL_UNIX_INSTALL_PATH):
-        AFL_UNIX_REPO = "https://github.com/shellphish/afl-other-arch"
+        AFL_UNIX_REPO = "https://github.com/mirrorer/afl"
         if subprocess.call(['git', 'clone', AFL_UNIX_REPO, AFL_UNIX_INSTALL_PATH]) != 0:
```
Comment (zardus) [inline setup.py]: Could you clone this with --depth=0?

File: patches/other-arch.diff [hunk 2]
```diff
@@ -0,0 +1,186 @@
+Only in afl-other-arch: build.sh
+Only in afl: .gitignore
+diff -r afl/qemu_mode/build_qemu_support.sh afl-other-arch/qemu_mode/build_qemu_support.sh
```
Comment (zardus) [inline patches/other-arch.diff]: This seems to dumplicate `build_qemu_support.sh` above. Could you deduplicate (and check out the comments on the above script as well)?

File: patches/build_qemu.diff [hunk 3]
```diff
@@ -0,0 +1,147 @@
+--- qemu_mode/build_qemu_support.sh	2019-01-25 19:26:49.314879371 -0700
```
Comment (zardus) [inline patches/build_qemu.diff]: Are these invasive changes to the build process necessary? The diff reads like a replacement of the entire build script. Is it a case of the build script continuing to evolve in upstream AFL? Let's try to have minimal necessary changes here.

**Review by zardus — CHANGES_REQUESTED** [review]
Thank you! There are some comments I put. Could you check them out?

**Comment by g0kkk** [comment]
@zardus, have made the necessary changes that were asked

**Comment by zardus** [comment]
Thank you!

**Pull request merged**